boostsec/converter/sarif/mobsfscan_sast.py

```diff
@@ -0,0 +1,273 @@
+"""Boost SAST extensions."""
+from typing import Optional, TypedDict
+
+from .base import (
+    ReportingDescriptor,
+    ReportingDescriptorReference,
+    ReportingDescriptorRelationship,
+    Run,
+    ToolComponent,
+    ToolComponentReference,
+)
+from .boostsec import BoostFindingConfidence, BoostFindingSeverity
+
+BOOST_SAST_TAXONOMY_NAME = "boost/sast"
+BOOST_SAST_TAXONOMY_VERSION = "1.0.0"
+BOOST_SAST_TAXONOMY_ORG = "boostsecurity"
+
+
+class MobsfscanCweType(TypedDict):
+    """Represents a typed dict for a given CWE."""
+
+    cwe_id: str
+    cwe_title: str
+    severity: BoostFindingSeverity
+    confidence: BoostFindingConfidence
+
+
+DEFAULT_RULE = MobsfscanCweType(
```

Comment:
by leveraging the mitre curated list of CWEs , the CWE unknown won't be needed anymore since the fall back to unknown will be done on the server.

Reply:
What is the recommend change here?

Remove `DEFAULT_RULE` and `DEFAULT_RULE_KEY` ?

Reply:
that's right as well as the fallback code that use the default rule when the rule is not in the allowed list.